.changeset/giant-ducks-tickle.md

```diff
@@ -0,0 +1,11 @@
+---
+"@onflow/util-encode-key": minor
+"@onflow/util-invariant": minor
+"@onflow/util-template": minor
+"@onflow/util-address": minor
+"@onflow/util-logger": minor
+"@onflow/util-uid": minor
+"@onflow/rlp": minor
+---
+
+Convert to Typescript
```

Comment:
We should probably try to consolidate all of the changesets

Reply:
these changesets should be consolidated to just one version bump, I think